releasenotes/notes/add-back-lost-expvar-endpoint-7f801a71c1856cda.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
```

Comment:
Let's wait for @hush-hush to confirm if we will be releasing this as part of another RC within 6.12.0. In that case, we should remove the change log because this bug was never released. Otherwise, we can leave it.